operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -157,34 +160,33 @@ private static Step readExistingPods(DomainPresenceInfo info) {
   // pre-conditions: DomainPresenceInfo SPI
   // "principal"
   static Step bringAdminServerUp(
-        DomainPresenceInfo info, PodAwaiterStepFactory podAwaiterStepFactory, Step next) {
-    return Step.chain(bringAdminServerUpSteps(info, podAwaiterStepFactory, next));
+        DomainPresenceInfo info, PodAwaiterStepFactory podAwaiterStepFactory) {
```

Comment:
is it ok to remove `next`?

Reply:
When it's in a call to Step.chain(), yes. That links the specified steps together, allowing them to be listed sequentially and not nested.